python/tvm/contrib/cc.py

```diff
@@ -373,7 +373,11 @@ def _linux_compile(
 
 def _windows_compile(output, objects, options, cwd=None, ccache_env=None):
     cmd = ["clang"]
+    compiler = os.getenv("TVM_WIN_CC", default="clang")
+    win_target = os.getenv("TVM_WIN_TARGET", default="x86_64")
+    cmd = [compiler]
```

Comment:
This overwrite initial ```cmd = [clang]``` initialization, in a counterintuitive logic, could we condition the new assignment here to ```TVM_WIN_TARGET``` presence ?

Reply:
Also lets fix ```cwd=None``` typo here, perhaps we could have ```cmd=[clang]``` (if you would like better) and then just override with windows env related conditional.

Reply:
@cbalint13   make sense, we don't require this assignment as we are consuming from environment variables.

Reply:
This is fine too (removing), I am sure the original would worked but its logic was strange. 

Thank you !

 Apollogise for confusion of ```cwd``` vs. ```cmd``` function parameter, it is not the case to involve that too.